packages/now-cli/src/util/link/setup-and-link.ts

```diff
@@ -39,9 +37,10 @@ export default async function setupAndLink(
 ): Promise<ProjectLinkResult> {
   const {
     authConfig: { token },
+    apiUrl,
+    output,
     config,
   } = ctx;
-  const { apiUrl } = ctx;
   const debug = output.isDebugEnabled();
```

Comment:
I think we can remove a bunch of debug args parsing too if we utilize this `output.isDebugEnabled()` everywhere

Reply:
Agreed, but should be a follow-up PR IMO.